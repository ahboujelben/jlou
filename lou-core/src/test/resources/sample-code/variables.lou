var str = "text";
print "this is a " + str;

var a;
print a;

a = 1;
print a;

print "Scoping";

{
    var a = 2;
    print a;

    var b = 3;
    print a + b;
}

print a;
print b;
