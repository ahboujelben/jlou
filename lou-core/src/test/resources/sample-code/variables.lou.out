this is a text
nil
1
Scoping
2
5
1
[line 21] Error at 'b': Undefined variable 'b'.